cwms-data-api/src/test/java/fixtures/TestAccounts.java

```diff
@@ -8,7 +8,8 @@ public class TestAccounts {
     public enum KeyUser {
         NONE("none",null,null,null,null), // Used for annotations
         GUEST("guest",null,null,null,null), // USED as marker label for tests
-        SPK_NORMAL("l2hectest","l2hectest","1234567890","l2userkey","ATotallyRandomString","CWMS Users", "cac_user"),
+        SPK_NORMAL("l2hectest","l2hectest","1234567890","byp","ATotallyRandomString","CWMS Users", "cac_user"),
```

Comment:
why was this changed?

Reply:
(the adding is fine but we shouldn't existing values without reason)

Reply:
I only have access to swt so I needed to create an swt user for my tests

Reply:
Okay, that happens. please revert it back.

Not having docker doesn't affect those permissions. You can just add users to the database; they don't need to be oracle users.

For clarity, are you attempting to do development work in a shared database or does you're IT team at least give you one per dev?

Reply:
Ok I can add the user to my db. I tested everything with swt but changed the tests back to spk to maintain uniformity throughout the tests. The db is one that RMA made for me, I know that Adam and Ryan have access to it as they've helped me debug.

Reply:
api key reverted

Reply:
Devin and Zac have their own separate databases that can get trashed and recreated whenever it is convenient for them to do so.  But they do persist outside the lifetime of the test-container